docs/notebooks/nmf_tutorial.ipynb

```diff
@@ -0,0 +1,1669 @@
+{
+ "cells": [
+  {
```

Comment:
Still missing/need to fix
- what we have in NMF in addition (normalization for using it as TM), 
- "Training" header duplicated
- Can you be a bit more "descriptive" (not only headers, also text descriptions)
- Benchmark require some conclusions (not just show a table, here you have many metrics & many parameters)
- Parameter tuning
- What happens with gensim NMF faces in notebook?